apps/data-feeds-config-generator/src/data-services/fetchers/exchanges/crypto-com-exchange/crypto-com-exchange.ts

```diff
@@ -13,16 +15,27 @@ export class CryptoComAssetsFetcher
   implements ExchangeAssetsFetcher<CryptoComAssetInfo>
 {
   async fetchAssets(): Promise<AssetInfo<CryptoComAssetInfo>[]> {
-    const assets = await fetchCryptoComInfo();
-    return assets.result.data.map(asset => ({
-      pair: {
-        base: asset.base_ccy,
-        quote: asset.quote_ccy,
-      },
-      data: {
-        instrument_name: asset.symbol,
-      },
-    }));
+    const assets = (await fetchCryptoComInfo()).result.data.filter(
+      asset => asset.inst_type === 'CCY_PAIR',
```

Comment:
This is a good improvement.

Soon we will be able to remove this https://github.com/blocksense-network/blocksense/blob/main/apps/oracles/crypto-price-feeds/src/exchanges/crypto_com_exchange.rs#L55

> soon = once we update the oracle arguments in the feeds config